lib/mayaUsd/ufe/StagesSubject.cpp

```diff
@@ -223,13 +223,11 @@ void StagesSubject::stageChanged(UsdNotice::ObjectsChanged const& notice, UsdSta
 			{
 				if (prim.IsActive())
 				{
-					auto notification = Ufe::ObjectAdd(sceneItem);
-					Ufe::Scene::notifyObjectAdd(notification);
```

Comment:
I believe that all your changes will need to keep this old code around and protect your new code with UFE preview version check. Something like this:

```
#if UFE_PREVIEW_VERSION_NUM >= 2021
    <insert new code here>
#else
    <keep old code here>
#endif
```

Reply:
addressed f4f6bd4a848585dfb332e3ee75ac98923af250b5